lib/connector/OAuthConnector.js

```diff
@@ -272,19 +272,29 @@ class OAuthConnector {
    * Deletes user context from storage.
    * @param {FusebitContext} fusebitContext The Fusebit context
    * @param {string} vendorUserId The vendor user id
-   * @param {string} foreignVendorId If specified, vendorUserId represents the identity of the user in another system.
-   * The foreignVendorId must correspond to an entry in userContext.foreignOAuthIdentities.
+   * @param {string} vendorId If specified, vendorUserId represents the identity of the user in another system.
```

Comment:
Extra unclear what vendor refers to now that there’s both vendorUserId and vendorId representing the identity of the user...